### PR TITLE
Build /automotive/resources page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Build /automotive/resources page based on [copydoc](https://docs.google.com/document/d/17ofQfYbMlTsjdpB3JJru5mckGeLFXMAxRbHvIL4TrOE/edit#heading=h.7etr2bdzumur) and [figma design](https://www.figma.com/file/u8mPLR8FH2j7p3YCxQBQQe/23.10-U.com---%2Fautomotive%2Fresources-(WD-4060)?type=design&node-id=0-1&mode=design&t=5FYZ6uwGduKHxLql-0)

## QA

- Go to https://ubuntu-com-13276.demos.haus/automotive/resources and compare against the copydoc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4062

### CODE REVIEW

**Comment by petesfrench** [comment]
Thanks @lyubomir-popov! I will look into it, but it might end up as a separate piece of work which we shouldn't let block this going live

**Comment by petesfrench** [comment]
@juanruitina Can you take a quick look please

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench, LGTM! There are a couple of things missing that are on the doc, the "Embrace a software-defined future" heading and the "Power up your transformation with trusted open source" CTA, but these are also not on the design so I'm not sure if the doc is just out of date

File: templates/automotive/resources.html [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+      <div class="row p-section--shallow" id="automotive-resources-latest-articles">
+      </div>
+    </div>
+  </div>
+
+  <template style="display:none" id="automotive-resources-articles-template">
+    <div class="col-3">
+      <div class="u-crop--16-9">
+        <a class="article-link" aria-hidden="true" tabindex="-1">
+          <div class="article-image"></div>
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/automotive/resources.html:233]: @lyubomir-popov It looks like the changes you made have introduced variable spacing above the blog images:
image
image
Which one did you want implemented?

**Comment by petesfrench** [comment]
> Thanks @petesfrench, LGTM! There are a couple of things missing that are on the doc, the "Embrace a software-defined future" heading and the "Power up your transformation with trusted open source" CTA, but these are also not on the design so I'm not sure if the doc is just out of date

@lyubomir-popov Were these headings omitted purposefully?

File: templates/automotive/resources.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (lyubomir-popov) [reply templates/automotive/resources.html:233]: that was intentional

[the remark this answers is not in the document]
Comment (lyubomir-popov) [reply templates/automotive/resources.html:233]: images shouldn't touch the hrs, so we use this class on them.

**Comment by lyubomir-popov** [comment]
> Were these headings omitted purposefully?

@juanruitina can you please confirm whether these headings are needed?


@petesfrench what was the outcome of the cdn retina issue? If we can't fix it, can we please stop using it and jus embed the image directly? And possibly start a bigger discussion about finding a cdn service thast is fit for retina screens? 

The image here is still blurry I'm affraid:
image

**Comment by juanruitina** [comment]
Took me some time but here I am :)

- [ ] Please add the "Power up your transformation with trusted open source" CTA (it was missing so I just incorporated it to the [mockup](https://www.figma.com/file/u8mPLR8FH2j7p3YCxQBQQe/23.10-U.com---%2Fautomotive%2Fresources-(WD-4060)?type=design&node-id=0%3A1&mode=design&t=siz5KchKZfsDXJp0-1) and checked with Lyubo).
- [ ] Bit of visual feedback, some headings ("Webinar", "From our blog" and "Latest") and the thumbnails under latest are lacking padding on top. (Paddings in the "In the media" section look good)
- [ ] Similarly, can you please add some spacing on mobile between the three items under "Whitepapers"?

I adjusted the copy doc to reflect the changes in the design; no further content changes are necessary other than the missing CTA section.

**Comment by lyubomir-popov** [comment]
> * some headings ("Webinar", "From our blog" and "Latest") and the thumbnails under latest are lacking padding on top.

missed that, thanks @juanruitina - padding on headings should not be removed, the u-no-padding class is not necessary here:
image

**Comment by juanruitina** [comment]
LGTM, thank you for the changes Pete!

**Comment by petesfrench** [comment]
@lyubomir-popov The image quality drops even when embed the image directly, making me think it is a problem with the image itself

**Comment by lyubomir-popov** [comment]
you aree right, sorry - here it is in 3x resolution https://assets.ubuntu.com/v1/8b1151ab-Artboard%201@3x.png

**Pull request merged**